.github/workflows/trigger.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           # is not a PR event
           REF=$(curl -sSf \
             --url ${{ steps.pr-standardize-comment.outputs.url }} \
-            --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+            --header 'Authorization: Bearer ${{ secrets.SCALE_GITHUB_TOKEN }}' \
```

Comment:
I cant test this further until it lands in master for the PR comment trigger to gitlab-ci, will do a follow up after we merge this though